src/Illuminate/Queue/QueueManager.php

```diff
@@ -295,6 +295,88 @@ public function setApplication($app)
         return $this;
     }
 
+    /**
+     * Pause a queue by name.
+     *
+     * @param  string  $queue
+     * @param  int  $ttl
+     * @return void
+     */
+    public function pause($queue, $ttl = 86400)
```

Comment:
Very strange solution to pause just for one day. Why not week, year, or 30 seconds? It's better to pass null as default, meaning by default pausing queue until it's resumed manualy.

For other cases developers still will have a way to pass any ttl they want in certain case